velox/common/compression/CMakeLists.txt

```diff
@@ -13,4 +13,4 @@
 # limitations under the License.
 
 add_library(velox_common_compression Compression.cpp LzoDecompressor.cpp)
-target_link_libraries(velox_common_compression Folly::folly)
+target_link_libraries(velox_common_compression velox_common_base velox_dwio_dwrf_proto velox_dwio_dwrf_common gtest Folly::folly)
```

Comment:
The idea of this PR is great, but can we do it cleanly and step by step?
1. Decouple compression and `PagedInputStream` with `DWRF` dependencies
2. Use it in parquet `PageReader`

Reply:
Do you suggest in two commits or two different PRs?

Reply:
2 different PR would be more easier to review